core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1119,6 +1122,7 @@ object KafkaConfig {
       .define(GroupCoordinatorNumThreadsProp, INT, Defaults.GROUP_COORDINATOR_NUM_THREADS, atLeast(1), MEDIUM, GroupCoordinatorNumThreadsDoc)
       // Internal configuration used by integration and system tests.
       .defineInternal(NewGroupCoordinatorEnableProp, BOOLEAN, Defaults.NEW_GROUP_COORDINATOR_ENABLE, null, MEDIUM, NewGroupCoordinatorEnableDoc)
+      .define(GroupProtocolMigrationPolicyProp, STRING, Defaults.GROUP_PROTOCOL_MIGRATION, in(Utils.enumOptions(classOf[GroupProtocolMigrationPolicy]):_*), MEDIUM, GroupProtocolMigrationPolicyDoc)
```

Comment:
Let's move it to the consumer group section. I think that we should also keep it internal (with defineInternal) for now.